### PR TITLE
Generate question service for creating data requests

### PR DESCRIPTION
**Pull request by CampbellBrett**
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- I checked the documentation and found no answer.
- I checked to make sure that this issue has not already been filed.
- I'm reporting the issue to the correct repository (for multi-repository projects).
- I've read the [CONTRIBUTING](https://github.com/HackCapital/cto.ai/blob/master/.github/CONTRIBUTING.md) document.
- If it is SRED eligible, I've added the label SRED.
- I've named a clear and solution focused title.

# Overview

A question service needs to be added so that users can request data

# Expected Behavior

As a developer I can manage questions in a postgresql database

# Current Behavior

Currently there is no postgresql database connected or question service.

# Tasks

- [x] Generate a question service
- [x] add initials questions columns to table
- [x] Post question to postgres through service using curl

# Additional Notes

(This is an optional section)
Use this sections to **add notes, reference other related issues/PR, or data such as benchmarks**

# Failure Case

**If it is not a bug, please remove the rest of this template.**
Please help provide information about the failure if this is a bug.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Step 1
2. Step 2
3. You get it...

## Context

Please provide any relevant information about your setup, **if relevant**.
This is important in case the issue is not reproducible except for under certain conditions.
Example: Operating System, Operating System version, and Runtime (NodeJS/Go/Rust) version.

## Logs

If any, please include relevant log snippets or files here.

### CODE REVIEW

File: src/models/questions.model.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        table.dateTime('updated_at');
+      })
+        .then(() => console.log(`Created ${tableName} table`))
+        .catch(e => console.error(`Error creating ${tableName} table`, e));
+    }
+  });
+
+
+  return db;
+}
```
Comment (slaterb1) [inline src/models/questions.model.js]: Did you talk with Thales about pulling the table initialization out of the .model file?

File: src/models/questions.model.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+
+// questions-model.js - A KnexJS
+// See http://knexjs.org/
+// for more of what you can do here.
+export default function (app) {
```
Comment (slaterb1) [inline src/models/questions.model.js]: more of a styling thing, but should we specify the name of this function? 
```javascript
export default function createQuestionModel(app) {
```

File: package.json [hunk 3]
```diff
@@ -24,23 +24,31 @@
   "scripts": {
     "test": "yarn run eslint && yarn run coverage",
     "eslint": "eslint src/. test/. --config .eslintrc.json",
-    "start": "node src/",
     "mocha": "mocha test/ --recursive --exit",
-    "coverage": "NODE_ENV=testing istanbul cover node_modules/mocha/bin/_mocha -- $(find test -name '*.test.js') --require babel-core/register --recursive --exit"
+    "coverage": "NODE_ENV=testing istanbul cover node_modules/mocha/bin/_mocha -- $(find test -name '*.test.js') --require babel-core/register --recursive --exit",
+    "start": "babel-node src/index.js | ./node_modules/.bin/bunyan -o short",
+    "docker": "NODE_ENV=docker yarn start"
```
[the remark this answers is not in the document]
Comment (CampbellBrett) [reply package.json]: `start` will work for local development as all it's configs are defined in default.json

File: src/models/questions.model.js (hunk 1 above)
Comment (CampbellBrett) [reply src/models/questions.model.js]: Haven't yet. Going to try and get this in first and handle that as a separate PR

File: src/models/questions.model.js (hunk 2 above)
Comment (CampbellBrett) [reply src/models/questions.model.js]: Definitely can to clarify what the function does.

**Pull request merged**